modules/tinymce/src/themes/silver/main/ts/ui/button/MenuButton.ts

```diff
@@ -9,9 +9,10 @@ import { UiFactoryBackstage } from '../../backstage/Backstage';
 import { renderCommonDropdown } from '../dropdown/CommonDropdown';
 import ItemResponse from '../menus/item/ItemResponse';
 import * as NestedMenus from '../menus/menu/NestedMenus';
+import { getSearchPattern } from '../menus/menu/searchable/SearchableMenu';
 import { ToolbarButtonClasses } from '../toolbar/button/ButtonClasses';
 
-export type MenuButtonSpec = Omit<Toolbar.ToolbarMenuButton, 'type'>;
+export type ToolbarMenuButtonWithoutType = Omit<Toolbar.ToolbarMenuButton, 'type'>;
```

Comment:
Primarily, because it was a post-boulder object, but it had `Spec` as a suffix in its name. Whereas in bridge, the `*Spec` type seemed to nearly always be the pre-boulder object

Reply:
- [ ] switch back.

Reply:
Yeah that's fair... we probably need to do this for all of them though if we're going to make a change like this instead of just for a single one.